.deploy-onto-config.yml

```diff
@@ -0,0 +1,5 @@
+# Configuration file for deploying ontologies on onto-ns.com
+ontology_dirs: ["."]
+avoid_namespaces: []
+catalog_file_dir: "."
```

Comment:
Why is there only one `catalog_file_dir` but multiple `ontology_dirs`?

Reply:
Because I expect there to only be a single `catalog-v001.xml` file - is that wrong?
The multiple-valued `ontology_dirs` is to be able to list multiple directories within the repository, from where ontology files should be retrieved and published. Hmm, maybe I just add some minor comments to each config value?